README.md

```diff
@@ -180,12 +189,24 @@ npm run deploy
 ## 👥 Authors <a name="authors"></a>
 
 
+
+
+👤 **Maher Ramadan**
+
+- GitHub: [@maherramadan78](https://github.com/maherramadan78/leaderboard)
+- 
+=======
+
+
+
```

Comment:
- [ ] This link is the link of the this project repo. here in the `Author` section, kindly consider adding your professional social media links like `linkedIn` and `Twitter` to make possible for the visitors to contact you.